Tests/ASCollectionViewTests.mm

```diff
@@ -1049,6 +1049,7 @@ - (void)testInitialRangeBounds
   UIWindow *window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
   ASCollectionViewTestController *testController = [[ASCollectionViewTestController alloc] initWithNibName:nil bundle:nil];
   ASCollectionNode *cn = testController.collectionNode;
+  cn.cellLayoutMode = ASCellLayoutModeAlwaysAsync;
```

Comment:
Shouldn't we test the default behavior? Maybe add another test case if need to for this new mode.

Reply:
Good call! Added a test.